pilz_control/test/unittest_pilz_joint_trajectory_controller.cpp

```diff
@@ -118,9 +120,27 @@ testing::AssertionResult PilzJointTrajectoryControllerTest::isControllerInUnhold
   return testing::AssertionSuccess();
 }
 
-///////////////////////////////////////
-//    The actual tests start here    //
-///////////////////////////////////////
+//////////////////////////////////
+//  Testing of Initialization   //
+//////////////////////////////////
+
+/**
+ * @brief Test if test environemt is initialized successfully.
+ */
+TEST_F(PilzJointTrajectoryControllerTest, testInitializiation)
```

Comment:
Tests can be shuffled like described by the [gtest framework](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#shuffling-the-tests), so what purpose does this test fulfill, if the tests are executed in random order?

Reply:
The comment is wrong. The `init()` method of the controller is tested here. I remove the comment.